### DISCUSSION TITLE
How can I set a default control_after_generate_value for INT type seed?

### DISCUSSION THREAD

**Discussion by MohammadAboulEla** [comfyanonymous/ComfyUI#4362] [Q&A]
how can I set a default control_after_generate_value for INT type
I mean like in none working code 
"seed": ("INT", {"default": 0,"control_after_generate_value":"fixed", "min": 0, "max": 0xffff})

**Comment by ltdrdata** [comment]
At present, the only method available is direct control through front-end code.
However, in what situations would you need to individually configure this value for each node?

Perhaps what's needed is a feature in the front-end to temporarily disable the control_after/before_generate functionality for all nodes?
